ros2interface/ros2interface/verb/show.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2interface.api import get_interface_path
+from ros2interface.api import type_completer
+from ros2interface.verb import VerbExtension
+
+
+class ShowVerb(VerbExtension):
+    """Output the interface definition."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+                'type',
+                help='Show an interface definition (e.g. "std_msgs/msg/String")')
+        arg.completer = type_completer
+
+    def main(self, *, args):
+        # TODO(kucheria) this logic should come from a rosidl related package
+        try:
+            parts = args.type.split('/')
+            if len(parts) < 2:
+                raise ValueError()
+            if not all(parts):
+                raise ValueError()
+            path = get_interface_path(parts)
+        except ValueError:
+            raise RuntimeError('The passed interface type is invalid')
+        except LookupError as e:
+            return str(e)
+        with open(path, 'r') as h:
```

Comment:
If the path does not exist, we should print a nicer error message. Right now we get an ugly stack trace:

```
$ ros2 interface show std_msgs/msg/Foo                                                                      1
Traceback (most recent call last):
  File "/home/jacob/ws/ros2cli_ws/install/ros2cli/bin/ros2", line 11, in <module>
    load_entry_point('ros2cli==0.7.0', 'console_scripts', 'ros2')()
  File "/home/jacob/ws/ros2cli_ws/install/ros2cli/lib/python3.6/site-packages/ros2cli/cli.py", line 69, in main
    rc = extension.main(parser=parser, args=args)
  File "/home/jacob/ws/ros2cli_ws/install/ros2interface/lib/python3.6/site-packages/ros2interface/command/interface.py", line 35, in main
    return extension.main(args=args)
  File "/home/jacob/ws/ros2cli_ws/install/ros2interface/lib/python3.6/site-packages/ros2interface/verb/show.py", line 42, in main
    with open(path, 'r') as h:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/ros/dashing/share/std_msgs/msg/Foo.idl'
```

I would first check if the file exists and return a nice error message if it does not, e.g. `return 'Could not find the interface {!r}'.format('/'.join(parts)))`.